src/sentry/search/eap/columns.py

```diff
@@ -102,25 +105,31 @@ class VirtualColumnDefinition:
 
 
 @dataclass(frozen=True, kw_only=True)
-class ResolvedFormula(ResolvedAttribute):
-    formula: Column.BinaryFormula
+class ResolvedFunction(ResolvedAttribute):
```

Comment:
Can you add a docstring here similar to your PR description differentiating the Formula & Aggregate thing?

Reply:
done!